src/components/SwitchCheckbox/SwitchCheckbox.tsx

```diff
@@ -73,6 +74,7 @@ const SwitchCheckbox = (props: Props) => {
     <div className={theme.outerWrap}>
 
       <label htmlFor={names.falseRadio} className={theme.falseRadio}>
+        <VisuallyHidden>False switch</VisuallyHidden>
```

Comment:
@Farhan-emgage let's make this "smarter" so that it uses the text "True, Null, False" when switchType prop is 'trueFalse' and "Yes, Null, No" switchType prop is 'yesNo' or 'normal'.

Reply:
@weegrom  the requested changes are done